api/src/main/scala/hmda/api/HmdaApi.scala

```diff
@@ -36,9 +41,27 @@ object HmdaApi
 
   //Start up API Actors
 
-  createSingleLarValidator(system)
+  val supervisor = HmdaSupervisor.createSupervisor(system)
+
+  (supervisor ? FindActorByName(SingleLarValidation.name))
+    .mapTo[ActorRef]
+    .map { actor =>
+      log.info(s"Started validator at ${actor.path}")
```

Comment:
We are already logging something very similar when the actors start up in the `HmdaActor` class: `"Actor started at ${self.path}"`. Do we want to log this twice?